pkg/datapath/iptables/iptables.go

```diff
@@ -494,6 +494,32 @@ func (m *IptablesManager) InstallRules(ifName string) error {
 			return err
 		}
 
+		// kube-proxy does not change the default policy of the FORWARD chain
+		// which means that while packets to services are properly DNAT'ed,
+		// they are later dropped in the FORWARD chain. The issue has been
+		// resolved in #52569 and will be fixed in k8s >= 1.8. The following
```

Comment:
This comment refers to the old issue, but this rule is necessary in general as long as iptables is present on the machine, no?